src/plugins/plot/axis/YAxis.vue

```diff
@@ -107,23 +102,71 @@ export default {
         }
     },
     mounted() {
-        this.yAxis = this.getYAxisFromConfig();
+        eventHelpers.extend(this);
+        this.initAxisAndSeriesConfig();
         this.loaded = true;
         this.setUpYAxisOptions();
     },
     methods: {
-        getYAxisFromConfig() {
+        initAxisAndSeriesConfig() {
             const configId = this.openmct.objects.makeKeyString(this.domainObject.identifier);
             let config = configStore.get(configId);
             if (config) {
-                return config.yAxis;
+                if (!this.id || this.id === config.yAxis.id) {
+                    this.yAxis = config.yAxis;
+                } else {
+                    this.yAxis = config.additionalYAxes.find(yAxis => yAxis.id === this.id);
+                }
```

Comment:
Nitpick, but I think this becomes easier to read if we invert the condition:
```suggestion
                if (this.id && this.id !== config.yAxis.id) {
                    this.yAxis = config.additionalYAxes.find(yAxis => yAxis.id === this.id);
                } else {
                    this.yAxis = config.yAxis;
                }
```

Reply:
Agreed. Done.